catalog/spatial/wfs/1.0.0/spatial-wfs-v1_0_0-source/src/main/java/org/codice/ddf/spatial/ogc/wfs/v1_0_0/catalog/source/WfsFilterDelegate.java

```diff
@@ -1116,6 +1102,20 @@ private FilterType buildGeospatialFilterType(String spatialOpType, String proper
             return buildBinarySpatialOpType(filterObjectFactory.createWithin(new BinarySpatialOpType()),
                     propertyName,
                     wkt);
+        case Beyond:
```

Comment:
What was the reason for reordering these?

Reply:
Logic was added to fix a resource leak/null pointer issue. The problem was that these cases no longer return or break if `distance == null` in these scenarios. The cases were moved down so that they would flow through each other and the default condition will be triggered instead of calling the return from another switch case.